blackbox/docs/sql/aggregation.txt

```diff
@@ -330,6 +330,56 @@ Example::
     Due to java double precision arithmetic it is possible that any two executions
     of the aggregation function on the same data produce slightly differing results.
 
+percentile
+==========
+
+The ``percentile`` aggregation function computes a `Percentile`_ over numeric
+non-null values in a column.
+
+Percentiles show the point at which a certain percentage of observed values occur. 
+For example, the 98th percentile is the value which is greater than 98% of the observed values.
```

Comment:
there are different definitions about the percentile ... however i think this description does not describe the implementation  ... see tests